test/esinstall/import-nothing/import-nothing.test.js

```diff
@@ -0,0 +1,15 @@
+const {install} = require('../../../esinstall/lib');
+
+describe('error-no-dep-list', () => {
+  it('importing nothing', async () => {
+    expect(() =>
```

Comment:
I think that if you don't return this, the async test is ignored (or if it fails Jest gets mad that it failed after the test already succeeded).

Reply:
Ah you‘re right. `.rejects` does need that when running async inside `expect()`. Will add.